README.md

```diff
@@ -56,9 +55,7 @@ This role is supported on RHEL/CentOS-7, RHEL/CentOS-8 and Fedora distributions.
 - [`Rsyslog`](https://www.rsyslog.com/) - The logging role default log provider used for log processing.
 - [`Elasticsearch`](https://www.elastic.co/) - Elasticsearch is a distributed, search and analytic engine for all types of data. One of the supported outputs in the logging role.
 
-## Logging Configuration
```

Comment:
There is a link pointing this old header.
` 36 To satisfy such requirements, logging role introduced 3 primary variables `logging_inputs`, `l    ogging_outputs`, and `logging_flows`. The inputs are represented in the list of `logging_input    s` dictionary, the outputs are in the list of `logging_outputs` dictionary, and the relationsh    ip between them are defined as a list of `logging_flows` dictionary. The details are described     in [Logging Configuration](#logging-configuration).
`